packages/core/src/ceramic.ts

```diff
@@ -873,28 +873,28 @@ export class Ceramic implements CeramicApi {
       let stream
       try {
         stream = await promiseTimeout(
-          this.loadStream(streamId, { atTime: query.atTime }),
+          this.loadStream(streamId, { atTime: query.loadOpts?.atTime }),
```

Comment:
we should pass in the full loadOpts object here, not just the atTime.  That's the whole advantage of using loadOpts instead - now we can set more options than just atTime!